.envSample

```diff
@@ -7,3 +7,8 @@ DB_USER=
 DB_PASSWORD=
 DB_NAME=
 DB_HOST=
+PGSSLMODE=require
+SENDGRID_API_KEY=
+EMAIL=
+HOME_URL=
+APP_DEVURL=
```

Comment:
Are the newly added environmental variables added on Heroku? I believe we need to add thes variables on heroku also.

Reply:
OK....I will add them